parlai/tasks/opensubtitles/agents.py

```diff
@@ -40,3 +48,22 @@ def setup_data(self, path):
         if alternate:
             for i, e in enumerate(alternate):
                 yield e, i == 0
+
+
+# Cuts off after 100,000 examples
```

Comment:
move this comment to a docstring below the class name:
```
class SmallTeacher(HalfTeacher):
   """This version of opensubtitles only includes 100,000 dialogs."""
```

(same with the other comments)

This makes it so that the comment can automatically extracted into documentation for ParlAI.